nilearn/plotting/tests/test_img_plotting.py

```diff
@@ -445,3 +446,201 @@ def test_plot_prob_atlas():
                               threshold=0.2)
     # Testing the 4D plot prob atlas with contours
     display = plot_prob_atlas(img, view_type='continuous')
+
+
+def test_get_plot_stat_map_params_pos_neg():
+    # data with positive and negative range
+    affine = np.eye(4)
+    data = np.array([[-.5, 1., 0.],
```

Comment:
Given that the function is supposed to handle NaNs you could throw one or two NaNs in the matrix. The same applies for the other tests.

That could be done in a further PR too.